app/controllers/tasks_controller.rb

```diff
@@ -6,7 +6,7 @@ class TasksController < ApplicationController
 
   def index
     @tasks = tasks(params[:status])
-    @percent = calcualte_progression
+    @percent = ProgressionCalculator.call(closed_tasks, current_user.tasks)
```

Comment:
I think we can pass only `current_user` and then you can later extract `closed_tasks` and `tasks` from it.